src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-multisite-details/rgw-multisite-details.component.spec.ts

```diff
@@ -40,6 +41,6 @@ describe('RgwMultisiteDetailsComponent', () => {
 
   it('should display right title', () => {
     const span = debugElement.nativeElement.querySelector('.card-header');
-    expect(span.textContent).toBe('Topology Viewer');
+    expect(span.textContent).toBe('Topology Viewer ');
```

Comment:
Why this change?

Reply:
Updated, had some change in the html regarding this, adjusted the test now in a better way